### ISSUE TITLE
The Fluent localization system

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#1508]
Mozilla is migrating its translations to a new format named FTL, part of the [Fluent](https://projectfluent.org/) system. Unlike GNU's PO format, FTL allows translators to handle all plurals and genders correctly.

Unfortunately we can't easily migrate to FTL because it isn't supported by Weblate yet (<https://github.com/WeblateOrg/weblate/issues/1926>).

**Comment by revi** [comment]
Under this system I guess we will have to translate both singular/plural and male/female gender stuff? Korean doesn't make use of GENDER or PLURAL stuff at all and it would be frustrating if I have to do 4 strings when it's (in reality) 1 string.

**Comment by Changaco** [comment]
@revi No, it's the opposite. Fluent allows translating a single message in different ways, but it doesn't require it.

**Comment by Changaco** [comment]
Weblate 4.8, which was released a few days ago, adds support for the Fluent file format.

**Comment by comradekingu** [comment]
Weblate does handle genders and plurals correctly, for any of the locales.
https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html
Can also use https://docs.weblate.org/en/latest/devel/translations.html#variants-while-translating for it.

It is inelegant to have gendered verbs, when running with "other" is always possible.
It also hinges on specifying gender to begin with, which is extra effort for no benefit in software, with only downside from a security perspective.
Then again individual cases for that can be set up in GNU gettext if need be.
